src/structures/GuildAuditLogs.js

```diff
@@ -500,20 +501,19 @@ class GuildAuditLogsEntry {
           ),
         );
     } else if (targetType === Targets.INVITE) {
-      this.target = guild.members.fetch(guild.client.user.id).then(me => {
+      this.target = guild.members.fetch(guild.client.user.id).then(async me => {
         if (me.permissions.has(Permissions.FLAGS.MANAGE_GUILD)) {
           let change = this.changes.find(c => c.key === 'code');
           change = change.new ?? change.old;
-          return guild.invites.fetch().then(invites => {
-            this.target = invites.find(i => i.code === change) ?? null;
-          });
+          const invites = await guild.invites.fetch();
+          this.target = invites.find(i => i.code === change) ?? null;
         } else {
           this.target = this.changes.reduce((o, c) => {
             o[c.key] = c.new ?? c.old;
             return o;
           }, {});
-          return this.target;
         }
+        return this.target;
```

Comment:
Are you sure this works still?

Reply:
i suppose there isn't really any point returning it or assigning it, ill change this

Reply:
i think this section may need some refactoring, but thats probably something for another PR